src/touch-vtt.js

```diff
@@ -34,14 +37,43 @@ Hooks.once('init', () => {
   initDirectionalArrows()
   initMeasurementTemplateEraser()
   initWallTools()
-  initEasyTarget()
+
+  // This wrap gives us control over every MouseInteractionManager
+  wrapMethod('MouseInteractionManager.prototype.callback', async function (originalMethod, event, ...args) {
+  	
+  	// v12 only: ugly patch to fix annoying issue in v12 where a double-click that opens a sheet also sends one of the clicks to an active listener on the sheet.
+  	// For example, you open an actor sheet, if something clickable is under your finger (icon, action, ability, etc.) it will get wrongly clicked.
+  	// What we do here is delay the sheet rendering a little bit, and also dispatch a right click on the canvas to avoid a lingering drag select on the placeable.
+  	if (parseInt(game.version) >= 12) {
```

Comment:
Just one thing: This repo uses 2 spaces for indentation, not tabs. Could you change that?

Reply:
Oops, my bad, a few tabs slipped in here and there. I added a new commit to this PR to remove all of them.